highfidelity/hifi
# Enable bloom in zone properties

https://highfidelity.manuscript.com/f/cases/15034/Add-bloom-control-to-the-Zone-UI-in-Create

Test plan:
- Go to your sandbox and go to an area with some bright areas.
- Create a zone.  In the zone properties, set Bloom to ON.
- In Running Scripts, search for and run debugBloom.js.  Select "All".
- You should see something like this:
![bloom1](https://user-images.githubusercontent.com/7650116/43880264-8404a130-9b5c-11e8-978f-c0d661faeb62.png)
- As you increase and decrease the Bloom Intensity, the bottom right image should get brighter and darker, respectively.
- As you increase and decrease the Bloom Threshold, all four images should get darker and brighter, respectively.
- As you increase and decrease the Bloom Size, all four images should get more and less blurred, respectively.
